examples/guestbook/README.md

```diff
@@ -67,11 +69,17 @@ The example consists of:
 
 The web frontend interacts with the redis master via javascript redis API calls.
 
-**Note**:  If you are running this example on a [Google Container Engine](https://cloud.google.com/container-engine/) installation, see [this Container Engine guestbook walkthrough](https://cloud.google.com/container-engine/docs/tutorials/guestbook) instead. The basic concepts are the same, but the walkthrough is tailored to a Container Engine setup.
+**Note**:  If you are running this example on a [Google Container Engine](https://cloud.google.com/container-engine/) installation, see [this Google Container Engine guestbook walkthrough](https://cloud.google.com/container-engine/docs/tutorials/guestbook) instead. The basic concepts are the same, but the walkthrough is tailored to a Container Engine setup.
 
 ### Prerequisites
 
-This example requires a running Kubernetes cluster. See the [Getting Started guides](../../docs/getting-started-guides/) for how to get started. And follow the [Prerequisites](../../docs/user-guide/prereqs.md) to make sure your `kubectl` is ok. As noted above, if you have a Google Container Engine cluster set up, go [here](https://cloud.google.com/container-engine/docs/tutorials/guestbook) instead.
+This example requires a running Kubernetes cluster. First, check that kubectl is properly configured by getting the cluster state:
+
+```console
+$ kubectl cluster-info
+```
+
+If you see a url response, you are ready to go. If not, read the [Getting Started guides](../../docs/getting-started-guides/) for how to get started, and follow the [Prerequisites](../../docs/user-guide/prereqs.md) to install and configure `kubectl`. As noted above, if you have a Google Container Engine cluster set up, read [this example](https://cloud.google.com/container-engine/docs/tutorials/guestbook) instead.
```

Comment:
Out of the scope of this pr, but the getting started guide link is a pretty rough landing.  Can we at least do the redirect for the user so it goes to http://kubernetes.io/docs/getting-started-guides/?